apps/netsuite/models.py

```
@@ -678,103 +678,103 @@ class Meta:
-            employee_mapping = EmployeeMapping.objects.filter(
-                source_employee__value=expense_group.description.get('employee_email'),
-                workspace_id=expense_group.workspace_id
-            ).first()
-            if employee_mapping and employee_mapping.destination_employee:
-                class_id = employee_mapping.destination_employee.detail.get('class_id')
-        
-        if not class_id:
-            class_id = get_class_id_or_none(expense_group, lineitem)
-
-        department_id = get_department_id_or_none(expense_group, lineitem)
+            category = lineitem.category if (lineitem.category == lineitem.sub_category or lineitem.sub_category == None) else '{0} / {1}'.format(
```

Ruff (0.7.0)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)